snappy-tools/build.gradle

```diff
@@ -103,7 +103,9 @@ task docs(type: ScalaDoc) {
         !it.getPath().matches('.*/internal/.*') && !it.getPath().contains('com/gemstone/gemfire/cache/operations/')
       })
       if (project.plugins.hasPlugin('scala')) {
-        allSource.addAll(project.sourceSets.main.allScala)
+        allSource.addAll(project.sourceSets.main.allScala.findAll {
+          !it.getPath().matches('.*org/apache/spark/sql/execution/joins/HashedRelation.*') 
+        })
       }
```

Comment:
Spark ignores internal packages while generating scaldoc . See Unidoc.ignoreUndocumentedPackages. I dont know whey we decided to document all. Ideally we should also ignore internal classes. I will file a bug and track it in a JIRA